js/src/tab.js

```diff
@@ -240,6 +270,36 @@ class Tab {
  * ------------------------------------------------------------------------
  */
 
+EventHandler.on(window, Event.LOAD_DATA_API, () => {
+  const tablists = makeArray(SelectorEngine.find(Selector.TABLIST))
+  if (tablists.length === 0) {
+    return
+  }
+
+  // iterate over all found sets of tab lists
+  for (let i = 0; i < tablists.length; i++) {
+    const tabs = makeArray(SelectorEngine.find(Selector.DATA_TOGGLE, tablists[i]))
+    let selectedTabFound = false
+
+    // iterate over each tab in the tablist, make sure they have correct tabindex/aria-selected
+    for (let j = 0; j < tabs.length; j++) {
```

Comment:
just a suggestion but you can use `.forEach` too

Reply:
still old-school me, so not quite sure how to remunge the code...particularly since at the end I jump back to the very first tab in the array if no selected tab was found during the run.

Reply:
something like: 
```js
tabs.forEach((tab) => {
  if (tab.getAttribute('aria-selected') === 'true' && !selectedTabFound) {
    tab.setAttribute('tabindex', '0')
    selectedTabFound = true
  } else {
    tab.setAttribute('tabindex', '-1')
    tab.setAttribute('aria-selected', 'false')
  }
})
```

btw I added some changes, it's not needed to use `.hasAttribute` because if no attribute `.getAttribute` will return `null`.